dhttp/server.go

```diff
@@ -198,10 +198,10 @@ type ServerConfig struct {
 
 	// OnShutdown is an array of functions that are each called once when shutdown is initiated.
 	// Use this when hijacking connections; your OnShutdown should notify your hijacking Handler
-	// to that a graceful shutdown has been initiated, and your Handler should respond by
-	// closing any idle connections.  This is used instead of dcontext soft Context cancellation
-	// because the Context should very much still be fully alive for any in-progress requests on
-	// that connection, and not soft-canceled; this is even softer than a dcontext soft cancel.
+	// that a graceful shutdown has been initiated, and your Handler should respond by closing
+	// any idle connections.  This is used instead of dcontext soft Context cancellation because
+	// the Context should very much still be fully alive for any in-progress requests on that
+	// connection, and not soft-canceled; this is even softer than a dcontext soft cancel.
```

Comment:
The change here is "… notify your hijacking Handler to that …" → "… notify your hijacking Handler that …" (drop the "to"); the rest is that dropping that word caused the remaining lines to re-wrap.